### PR TITLE
add methods: url and getParam

### PR DESCRIPTION
**Pull request by AlexLarra**
## New methods

- `url()`
- `getParam(param)`

### CODE REVIEW

File: src/controller.js [hunk 1]
```diff
@@ -36,6 +36,21 @@ export default class Controller {
     Rails.fire(_element(query), 'submit')
   }
 
+  url() {
+    return window.location.href
+  }
+
+  urlParams() {
```
Comment (markets) [inline src/controller.js]: is this method really useful as public API?

Comment (AlexLarra) [reply src/controller.js]: Yes it probably would be better to change it to private

Comment (markets) [reply src/controller.js]: +1

File: README.md [hunk 2]
```diff
@@ -72,3 +72,6 @@ export default class FlashMessages {
 - removeClass
 - visit
 - submit
+- url
+- urlParams
+- getParam
```
Comment (markets) [inline README.md]: @AlexLarra could you please add here method signature? I think it helps a lot to understand what's going on under the hood

File: src/controller.js [hunk 3]
```diff
@@ -36,6 +36,19 @@ export default class Controller {
     Rails.fire(_element(query), 'submit')
   }
 
+  url() {
+    return window.location.href
+  }
+
+  getParam(param) {
+    let urlParams = new URL(url()).searchParams
```
Comment (markets) [inline src/controller.js]: const

**Pull request merged**